src/Goerli-DssSpell.t.sol

```diff
@@ -92,16 +92,10 @@ contract DssSpellTest is GoerliDssSpellTestBase {
             assertEq(spell.expiration(), spellValues.deployed_spell_created + spellValues.expiration_threshold, "TestError/spell-expiration");
 
             // If the spell is deployed compare the on-chain bytecode size with the generated bytecode size.
-            // extcodehash doesn't match because the code of DssExec is different on every deployment since it takes a timestamp as a constructor param.
-            // However, the code hash of the spell action should match.
+            // extcodehash doesn't match because of the attached metadata (see test_bytecode_matches for comparison without it).
             address depl_spell = spellValues.deployed_spell;
             address code_spell = address(new DssSpell());
-
             assertEq(getExtcodesize(depl_spell), getExtcodesize(code_spell), "TestError/spell-codesize");
```

Comment:
The code size check is not redundant as it's checking `DssSpell` bytecode length, and not only the `DssSpellAction`.
I did move it to the beginning of `test_bytecode_matches`, so that test can handle all bytecode related checks.

Reply:
Yes you are right, it's actually checking `DssSpell` and just directly not the `DssSpellAction`.
Moving to the other test makes sense.